.github/workflows/ci-build-image.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           issue-number: ${{ github.event.client_payload.github.payload.issue.number }}
           body: |
-            :test_tube: [CI has Started Image Build]( https://github.com/${{github.repository}}/actions/runs/${{github.run_id}})" }'
+            :test_tube: [CI has Started Image Build]( https://github.com/${{github.repository}}/actions/runs/${{github.run_id}})
```

Comment:
Do we need an another comment just to show it is building? Why not add an emoji to indicate the progress and done, then finally a comment to show the full url of the image?

Reply:
it already adds an emoji shall i remove this comment ?